src/utils/constants.ts

```diff
@@ -59,9 +59,11 @@ export const enum CMP_FLAGS {
   shadowDomEncapsulation = 1 << 0,
   scopedCssEncapsulation = 1 << 1,
   hasSlotRelocation = 1 << 2,
+  // TODO(STENCIL-662): Remove code related to deprecated shadowDomShim field
```

Comment:
No action required ATM. We will probably want to do something with regard to "reserving" this space in the future, even if it's something like:
```diff
- needsShadowDomShim = 1 << 3,
+ // the fourth bit once occupied a 'needsShadowDomShim' flag that was removed in Stencil X
```

So that if multiple versions of the runtime are implicitly mixed, we don't end up reusing that bit for something else (or shifting the ones that follow `needsShadowDomShim`). But maybe that doesn't matter if we declare different major versions of the runtime incompatible 🤷

Reply:
mmm that's a good thing to keep in mind...I'll add a note to this comment so that doesn't get lost